concepticondata/conceptlists.tsv

```diff
@@ -347,4 +347,5 @@ BirketSmith-1928-510	Birket-Smith, Kaj	1928		510	basic, areal, historical	Englis
 Chen-2019-61	Chen, I-Hsuan and Zhao, Qingqing and Long, Yunfei and Lu, Qin and Huang, Chu-Ren	2019		61	ratings	English	Chinese	https://figshare.com/articles/dataset/Mandarin_Chinese_modality_exclusivity_norms/7747022	Chen2019		The list presents modality exclusivity norms for 61 Chinese adjectives rated on visual, auditory, haptic, gustatory, and olfactory associations. The adjectives are two-character compounds.		
 Chen-2019-171	Chen, I-Hsuan and Zhao, Qingqing and Long, Yunfei and Lu, Qin and Huang, Chu-Ren	2019		171	ratings	English	Chinese	https://figshare.com/articles/dataset/Mandarin_Chinese_modality_exclusivity_norms/7747022	Chen2019		The list presents modality exclusivity norms for 171 Chinese adjectives rated on visual, auditory, haptic, gustatory, and olfactory associations. The adjectives are single-character words.		
 Speed-2017-485	Speed, Laura J. and Majid, Asifa	2017		485	ratings	English	Dutch	https://doi.org/10.3758/s13428-017-0852-3	Speed2017		The list presents modality exclusivity norms for 485 Dutch nouns rated on visual, auditory, haptic, gustatory, and olfactory associations. The authors also collected ratings of emotion (valence, dominance, and arousal) to assess its role in perceptual space simulation.		
+Gallatin-1836-180	Gallatin, Albert	1836		180	basic, areal, historical	English	Native languages in North America		Gallatin1836	Gallatin1836	This list was used in Gallatin’s groundbreaking ethnographic work on the native languages in North America, particularly in the areas west and north-west of the Rocky mountains. The selection process for items on the list was centered around the idea of showing the differences between the different native languages. As such, the glosses were build from basic vocabulary as well as very general concepts, thought to exist in the majority of languages under investigation.	307-367	
```

Comment:
Since there was not much to find in this list, let me be finicky here ;)
- particularly in the areas west and north-west of the Rocky mountains -> Rocky Mountains
- As such, the glosses were build from basic vocabulary -> were built

Reply:
Thanks!